react/index.js

```diff
@@ -238,36 +237,34 @@ class ProductSummary extends Component {
           {...showFieldsProps}
         />
       </div>
-    );
+    )
   }
 
   get renderBuyButton() {
-
     const {
       product,
-      displayMode,
       displayBuyButton,
       isOneClickBuy,
       buyButtonText,
       runtime: { hints: { mobile } },
     } = this.props
 
-    const buyButtonClasses = classNames(
-      `${productSummary.buyButtonContainer} pv3 w-100`,
-      {
-        'dn': displayMode === 'small' || displayMode === 'inline',
-        'dn db-ns': displayMode === 'normal',
-      }
-    )
+    const buyButtonClasses = classNames(`${productSummary.buyButtonContainer} pv3 w-100 db`)
```

Comment:
You don't need use `classNames` here, you can put it directly in the element's classname.